### ISSUE TITLE
Some subjects in document are seen as blank nodes

### ISSUE THREAD

**Issue by rubensworks** [rubensworks/jsonld-streaming-parser.js#33]
The subject for `http://www.w3.org/ns/ldp#contains` in https://api.coopstarter.happy-dev.fr/resources/ seems to be parsed as a blank node, while there is a valid `@id` present, so this should become a named node.

**Comment by rubensworks** [comment]
For reference, minimal example that produces incorrect parsing:

```json
{
  "@context": "https://cdn.happy-dev.fr/owl/hdcontext.jsonld",
  "@id": "https://api.coopstarter.happy-dev.fr/resources/",
  "ldp:contains": [
    {
      "@id": "https://api.coopstarter.happy-dev.fr/resources/1/",
      "@context": {
        "preview_image": "foaf:depiction"
      }
    }
  ]
}

```

**Comment by rubensworks** [comment]
Apparently I already came across this problem (and fixed it) in the 1.1 branch.
Cherry-picked the changes in, and released as 1.1.2.
